src-frontend/app/authorizers/token.js

```diff
@@ -0,0 +1,14 @@
+import Base from 'simple-auth/authorizers/base';
+
+export default Base.extend({
+    authorize: function(jqXHR, requestOptions) {
+        var session = this.get('session');
+
+        if (session.isAuthenticated) {
```

Comment:
A successful authentication in `app/authenticators/token.js` will cause this to be set to `true` automatically :)